build.gradle

```diff
@@ -30,6 +30,16 @@ buildscript {
 
 allprojects {
     repositories {
+        maven {
+            url "${rootDir}/.local-m2/"
+        }
+
+        // Supports artifact only dependencies like those from medicmobile repo
+        maven {
```

Comment:
This is a weird one. The medic mobile repo is "artifact only" - it just includes archives (like a JAR) instead of an additional `pom.xml` (or whatever else Maven might expect). By default, Gradle doesn't work with these kinds of repos and so you have to specify (with the `metaDataSources` block). The simplest way I found of getting it working with one repo was just to list the repo twice and have one configured as an "artifact only" source.

Reply:
Ahh I don't like it... if this is the best workaround that's fine but maybe we should ask the medicmobile team to publish the project to the maven repo so that we don't need to use such tricks?

Reply:
After digging into the Gradle docs a little more, it does look like we can [support both metadata sources in the one repo](https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:supported_metadata_sources) (it seems like they removed `artifact` as a default in Gradle 6.0) so I'll give that a try.

Interestingly, it does look like there was a discussion back in 2018 about moving Bikram Sambat to Maven Central between our very own @lognaturel and @alxndrsn. Would either of you have any updates on if there was a blocker on that?